### PR TITLE
fix: update instance dependency

### PR DESCRIPTION
**Pull request by Fine0830**
Reference  https://github.com/apache/skywalking/issues/6490

Screenshot

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: src/store/modules/topology/index.ts [hunk 1]
```diff
@@ -643,10 +643,15 @@ const actions: ActionTree<State, any> = {
         for (const call of topoCalls) {
           if (call.detectPoints.includes('CLIENT')) {
             clientIdsC.push(call.id);
-          } else {
+          }
+          if (call.detectPoints.includes('SERVER')) {
             serverIdsC.push(call.id);
           }
         }
+        if (!serverIdsC.length || !clientIdsC.length) {
```
Comment (wu-sheng) [inline src/store/modules/topology/index.ts:651]:
```suggestion
        if (!serverIdsC.length && !clientIdsC.length) {
```
Should use `&&`? And will you build request by the matrix of `serverIdsC` and `clientIdsC`?

Comment (Fine0830) [reply src/store/modules/topology/index.ts:651]: >  And will you build request by the matrix of `serverIdsC` and `clientIdsC`?

Yes

Comment (Fine0830) [reply src/store/modules/topology/index.ts:651]: > Should use `&&`

Here I need to change the logic.

**Comment by Fine0830** [comment]
Please recheck @wu-sheng

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**